frontend/webpack/base.js

```diff
@@ -19,6 +19,8 @@ const IN_WATCH_MODE = process.argv.includes("--watch");
 
 const BASE_DIR = path.resolve(path.join(__dirname, "..", ".."));
 
+const GIT_REVISION = process.env.GIT_REVISION || null;
```

Comment:
Note that this environment variable is ultimately set by [`deploy.py`](https://github.com/JustFixNYC/tenants2/blob/master/deploy.py), which retrieves the git hash and passes it into `Dockerfile.web` as a build argument:

https://github.com/JustFixNYC/tenants2/blob/1f1b9f314de2d488e1092e6b08da0c1f915af59f/Dockerfile.web#L50-L56